README.md

```diff
@@ -9,5 +9,17 @@ NaNMath.log(-100) # NaN
 NaNMath.pow(-1.5,2.3) # NaN
 ```
 
+In addition this package provides functions, that aggregate one dimensional arrays and ignore elements, that are NaN.
```

Comment:
Slight nitpick but I think the sentence would read better without the two commas here.